inline.md

```diff
@@ -0,0 +1,106 @@
+## paymentspring checkout form
+
+The checkout form is a simple way to seamlessly integrate the checkout button's ease of use natively into your site.
+
+The form elements are placed along side your html making the form part of your user experience and ultimately gives you the control.
+
+Each form is configurable with all the options found on our [button examples page](https://paymentspring.github.io/checkout-examples/).
+
+### Example
+
+To include a form on your website, you will need to obtain a form-id. Then copy and paste a line of javascript into your site where you would like the form to appear.
```

Comment:
I think that we will want to extract the shared copy into  [include](https://github.com/paymentspring/checkout-examples/tree/master/_includes) files. Another option would be to set up additional [layouts](https://github.com/paymentspring/checkout-examples/tree/master/_layouts), but that may be overkill with the number of differences between these pages.